.github/workflows/release-final.yml

```diff
@@ -17,7 +17,6 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: main
-          fetch-depth: 0
```

Comment:
We actually needed a `fetch-depth: 2` here so that we can compare the versions of LLD and LLM